src/bundle/Resources/public/js/scripts/admin.location.bookmark.js

```diff
@@ -15,14 +16,31 @@
     const isCurrentLocation = (locationId) => {
         return parseInt(locationId, 10) === currentLocationId;
     };
+    const setBookmarkWrapperCheckedClass = (bookmarked) => {
+        const wrapperClassList = document.querySelector(SELECTOR_BOOKMARK_WRAPPER).classList;
```

Comment:
Should be `const wrapper = document.querySelector(SELECTOR_BOOKMARK_WRAPPER);`

Reply:
`doc`